src/app/@auth/components/login/login.component.html

```diff
@@ -1 +1,40 @@
-<p>login works!</p>
+<h1 class="title">Welcome back</h1>
+<h2 class="subtitle">Continue track your progress</h2>
+
+<app-auth-error-message *ngIf="errorAfterLoading$ | async" [message]="errorMessage$ | async"></app-auth-error-message>
```

Comment:
we can use *ngIf and message directly in the component because it won't be reused in other places